src/pages/login.tsx

```diff
@@ -1,8 +1,38 @@
 import React, { useState } from "react";
+import { isValidEmail, isValidPassword } from "../utils/validation";
 
 const Login = () => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [emailError, setEmailError] = useState("");
+  const [passwordError, setPasswordError] = useState("");
+
+  const handleLogin = () => {
+    let Valid = true;
```

Comment:
```suggestion
    let valid = true;
```

Variáveis locais usamos `camelCase`